packages/core/types/src/promotion/http.ts

```diff
@@ -4,6 +4,7 @@ export type AdminGetPromotionRulesRes = {
   attribute_label: string
   field_type?: string
   operator: string
+  hydrate: boolean
```

Comment:
thought: Could I get you to write a brief doc on how all of these properties tie together and their function, specifically hydrate and disguised? I lost track of that over the past months 😅

Reply:
Yes will do, I think most of these can now be removed as we don't use them anymore. Will do a cleanup after all the kinks are sorted. 🤞🏻

Reply:
Great 💪